docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
This script is not used. I think it should be executed when docker command receives SIGTERM signal.

Reply:
https://github.com/prestosql/presto/blob/22c82faedd8576c7b574c4a7594f5461d8bb2b30/docker/bin/run-presto#L23
https://github.com/airlift/airlift/blob/9bdfeea74b5936ede0a6b153e220440700848ddf/launcher/src/main/scripts/bin/launcher#L39

The launch scripts are using `exec`. This means Presto JVM will become PID 1. It is not possible to trap the SIGTERM from launch script. Base on the discuss from https://github.com/prestodb/presto/issues/9456, handling SIGTERM from Presto JVM is not a good idea. This means this added script won't work with `docker kill`.

I add this script for users to either:
1. Execute the stop script explicitly using `docker exec`
```bash
$ docker exec -it <presto-worker-container> /usr/lib/presto/bin/stop-presto
```

2. Using the lifecycle hook from container orchestration system, such as [preStop](https://kubernetes.io/docs/tasks/configure-pod-container/attach-handler-lifecycle-event) from Kubernetes.
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: presto
spec:
  containers:
  - name: presto
    image: prestosql/presto
    lifecycle:
      preStop:
        exec:
          command: ["/bin/bash", "/usr/lib/presto/bin/stop-presto"]
```

Reply:
> The launch scripts are using exec. This means Presto JVM will become PID 1. It is not possible to trap the SIGTERM from launch script.

You could have a wrapper script that starts launcher as it's child

> Using the lifecycle hook from container orchestration system, such as preStop from Kubernetes.

That is specific to k8s. It would be great if it worked for plain Docker too.

I'm ok to leave it as is though.

Reply:
Implemented the requested feature in 57132ee. Please help review as I'm not familiarize with trap / pid system.